src/main/java/org/jitsi_modified/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -329,10 +332,6 @@ else if (
         if (updateEstimate)
         {
             lastUpdateMs = nowMs;
-            if (observer != null)
-            {
-                observer.onReceiveBitrateChanged(getSsrcs(), targetBitrateBps);
```

Comment:
has this moved to somewhere else?

Reply:
Yes, it's in `BandwidthEstimator.kt`.  The AbsSendTime estimator doesn't have its own callbacks anymore, it's just a component of `GoogleCcEstimator`.